msctl

```diff
@@ -2030,6 +2034,10 @@ OVERVIEWER_URL=$(getDefaultsValue 'mscs-overviewer-url' 'http://overviewer.org')
 MAPS_LOCATION=$(getDefaultsValue 'mscs-maps-location' $LOCATION'/maps')
 MAPS_URL=$(getDefaultsValue 'mscs-maps-url' 'http://minecraft.server.com/maps')
 
+# Grab the latest version information.
+# ---------------------------------------------------------------------------
+updateVersionsJSON
+
```

Comment:
I just audited the options that require that the `version_manifest.json` be up to date, and I narrowed it down to _start_, _update_, and _overviewer_.  We should move this down into those options to further reduce the need to run this function.